raft_test.go

```diff
@@ -4978,6 +4978,12 @@ func setRandomizedElectionTimeout(r *raft, v int) {
 	r.randomizedElectionTimeout = v
 }
 
+// SetRandomizedElectionTimeout is like setRandomizedElectionTimeout, but
+// exported for use by tests that are not in the raft package, using RawNode.
+func SetRandomizedElectionTimeout(r *RawNode, v int) {
+	setRandomizedElectionTimeout(r.raft, v)
+}
```

Comment:
Can we just change `setRandomizedElectionTimeout(r *raft, v int)` to `SetRandomizedElectionTimeout(r *raft, v int)`, and  not to add `SetRandomizedElectionTimeout(r *RawNode, v int)`?

Reply:
OK.